modules/apps/adaptive-media/adaptive-media-web/src/main/resources/META-INF/resources/adaptive_media/view.jsp

```diff
@@ -35,7 +35,10 @@ AMManagementToolbarDisplayContext amManagementToolbarDisplayContext = new AMMana
 PortletURL portletURL = renderResponse.createRenderURL();
 %>
 
-<div class="closed container-fluid-1280 sidenav-container sidenav-right" id="<portlet:namespace />infoPanelId">
+<clay:container
+	className="closed sidenav-container sidenav-right"
+	id="<portlet:namespace />infoPanelId"
```

Comment:
This won't work inside `tags`. It works inside `div` because the context allows for the rendering of `<portlet:namespace />`.

In the tag context, however, this needs to be rendered as in the other cases like:

```
id='<%= renderResponse.getNamespace() + "infoPanelId" %>`
```

I will take care of these cases when you send it to me.